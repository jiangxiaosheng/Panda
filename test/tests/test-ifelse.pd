func main() {
	var i = 0
    var p = true

    if i > 0 {
        i += 3

        if i == 0 {
            i += 2
        }
    }else{
        i += 1
    }
}