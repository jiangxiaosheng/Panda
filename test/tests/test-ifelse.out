

Sementically checked program: 


void main() {
var i: int(int : 0)
var p: bool(bool : true)
if ((bool : (int : i) > (int : 0))) {
(int : i += (int : 3))
if ((bool : (int : i) == (int : 0))) {
(int : i += (int : 2))
}
} else {
(int : i += (int : 1))
}
}

